test/e2e/nutanix_test.go

```diff
@@ -474,6 +564,116 @@ func TestNutanixKubernetes127SimpleFlowWithUUID(t *testing.T) {
 	runSimpleFlow(test)
 }
 
+func TestNutanixKubernetes128RedHatSimpleFlowWithUUID(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewNutanix(t, framework.WithRedHat128NutanixUUID(),
+			framework.WithPrismElementClusterUUID(),
+			framework.WithNutanixSubnetUUID()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube128)),
+	)
+	runSimpleFlow(test)
+}
```

Comment:
I dont think we need separate UUID tests since the images are going to be the same and it's not really adding a lot more coverage. If we really wanted, we can keep just the 128 tests for UUID and remove the other ones.